### DISCUSSION TITLE
UBL leveling

### DISCUSSION THREAD

**Discussion by btwo5** [hillsoftware/sv06#23] [Q&A]
after adding the g29 commands to prusa slicer my machine does all the 9 points,

however after reading the internet, would i be right in thinking i dont need m420 s1 ?

**Comment by eric-worth** [comment]
Correct, see hillsoftware's comment here: [Comment #9](https://github.com/hillsoftware/sv06/discussions/9#discussioncomment-5429068)

**Reply by btwo5** [reply]
great ive deleted, would i be right in thinking the m420 s1 would cancel the bed mesh ?

**Reply by eric-worth** [reply]
I can't say for sure, it just seems that m240 s1 is specific to Manual Mesh or Bi-Linear

**Comment by hillsoftware** [comment]
You do not need M420 s1 when using UBL
